packages/@lwc/engine/src/framework/main.ts

```diff
@@ -28,3 +28,5 @@ export { readonly } from './readonly';
 export { deprecatedBuildCustomElementConstructor as buildCustomElementConstructor } from './wc';
 
 export { setFeatureFlag, setFeatureFlagForTest } from '@lwc/features';
+
+export { registerTemplateSwap, registerComponentSwap, registerStyleSwap } from './hot-swaps';
```

Comment:
Bike shedding: What about just `swapTemplate`, `swapComponent` and `swapStyle`?

Reply:
Makes more sense to me.

Reply:
👍

Reply:
renamed